packages/block-editor/src/components/default-block-appender/index.js

```diff
@@ -53,14 +53,16 @@ export function DefaultBlockAppender( {
 			data-root-client-id={ rootClientId || '' }
 			className="wp-block block-editor-default-block-appender"
 		>
-			<TextareaAutosize
-				role="button"
-				aria-label={ __( 'Add block' ) }
-				className="block-editor-default-block-appender__content"
-				readOnly
-				onFocus={ onAppend }
-				value={ showPrompt ? value : '' }
-			/>
+			<p>
+				<TextareaAutosize
+					role="button"
+					aria-label={ __( 'Add block' ) }
+					className="block-editor-default-block-appender__content"
+					readOnly
+					onFocus={ onAppend }
+					value={ showPrompt ? value : '' }
+				/>
+			</p>
```

Comment:
Can't we render a freshly created default block?

Reply:
This "technically a button" appender goes way back to the earliest conversations, and I believe it's a button because it's not technically content yet, or making a new post instantly "dirty". I believe there was an accessibility aspect to it as well, which is why it had to be a button.